packages/media/src/generateVideoStrip.js

```diff
@@ -122,13 +140,16 @@ async function generateVideoStrip(element, resource, stripWidth, stripHeight) {
       actualFrameWidth,
       frameHeight,
       // Destination position
-      frame * actualStripFrameWidth,
+      dx,
```

Comment:
Hm, somehow it now doesn't correctly render the frame inside the strip - if you resize the original video in only one dimension (by squeezing it longer or thinner), the strip ends up with extra whitespace either above or below the images and the frames themselves aren't fully inside the strip:

| Before | After |
|-|-|
| ![flipped-video-trim-before](https://user-images.githubusercontent.com/637548/142699754-0c52f871-be4f-4de2-8bed-6e8dc8e8c51d.gif) | ![flipped-video-trim-after](https://user-images.githubusercontent.com/637548/142699748-7c9b5b7b-9014-4d45-bcad-3ba25509a287.gif) |

This particular video is flipped in both directions, but I get the same issue if only flipped in one direction (and resized in one dimension). No issue if not flipped and resized.

Reply:
@barklund  Good catch, didn't notice this at first. I was thinking it's because of the `ctx.translate()` but am seeing it also without any flip and on main as well. It might be related to the size of the video we are using to extract the frames. looking into it but in the meantime , could you confirm if you are not getting the same on main?

Reply:
@Trapsta I was testing it out as well (with the same video as Morten) and don't see the issue on `main` but can see it on this PR.
(In Morten's comment, the *Before* column of the table is from `main`.)

Reply:
@miina the issue was I was translating vertically using the width instead of the height, after fixing that am now getting the same thing am seeing on main which is a little strip of white and the frames in between
![Screenshot 2021-11-23 at 15 27 09](https://user-images.githubusercontent.com/276316/143024068-793dfd05-831b-41ca-b375-6c0186ef2a09.png)
00

Reply:
Which browser are you using? I can see everything correctly on `main`, the feature also has passed QA and bug-bash without anyone reporting this issue 🤔 

Could you create a story in the QA environment where this issue is visible and send the link to the story for checking?

Reply:
Am using chrome, was actually just discussing it with Morten -  try this story on QA https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/post.php?post=15287&action=edit

Reply:
@Trapsta If this still remains an issue after this PR, would you mind creating a new issue for fixing it?

Reply:
Cool, np

Reply:
@miina added https://github.com/google/web-stories-wp/issues/9870